src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java

```diff
@@ -47,34 +47,26 @@ public void submit(Runnable task) {
         executor.submit(task);
     }
 
-    public <T> void run(
-            Consumer<TaskParameters<T>> task,
-            Optional<T> initialLastResult,
-            Consumer<T> onResult) {
+    public <T> void run(Consumer<TaskParameters<T>> task, Optional<T> initialLastResult, Consumer<T> onResult) {
         task.accept(new TaskParameters(initialLastResult.get(), onResult));
     }
 
-    public <T> void submit(
-            Consumer<TaskParameters<T>> task,
-            Optional<T> initialLastResult,
-            Consumer<T> onResult) {
+    public <T> void submit(Consumer<TaskParameters<T>> task, Optional<T> initialLastResult, Consumer<T> onResult) {
         executor.submit(() -> task.accept(new TaskParameters(initialLastResult.get(), onResult)));
     }
 
-    public <T> void subscribe(
-            Subscription subscription,
-            Consumer<TaskParameters<T>> task,
-            Optional<T> initialLastResult,
-            Consumer<T> onResult,
-            BackOffRunnableConfig backOffRunnableConfig
-            ) {
+    public <T> void subscribe(Subscription subscription,
+                              Consumer<TaskParameters<T>> task,
+                              Optional<T> initialLastResult,
+                              Consumer<T> onResult,
+                              BackOffRunnableConfig backOffRunnableConfig) {
 
```

Comment:
Previous alignment is better when long names are used.
Alsthogh the final bracket should be in the same line.

Reply:
hmm what long names are you referring to?

Reply:
This I can do. There's an eclipse setting for that.